### ISSUE TITLE
[BUG] Do not double crop images for characters and personas.

### ISSUE THREAD

**Issue by Sqrlly** [SillyTavern/SillyTavern#1298]
### Environment

Self-Hosted (Bare Metal)

### System

Waterfox, Windows 10

### Version

SillyTavern 1.10.7 'release' (97b9d995)

### Describe the problem

For images that are not already in a 2:3 (400x600) size, either loaded manually or as a tavern card, they are first cropped into a 2:3 size.  Then when they are to be displayed as an avatar/persona image, the 2:3 cropped image is then cropped again into a square image, and then cropped into a circle.  See the example given in additional info.

This results in a much smaller (zoomed) view of the character than is necessary.  It would be desired to either keep the original aspect of the image and only crop as needed (such as into a circle).  Or if it *has* to be stored as a 2:3 image, expand the image with transparency as needed so when it's later cropped for display you get the largest view possible.  I could provide some logic if this route is required, but keeping the original aspect would be best.

With this, the manual loading would need to be modified, when the user is asked to select how they want to crop it.  I would propose rather than the one fixed 2:3 setting as it is, there be three:  Free (the default, the full image selected), 2:3, and Square.

### Additional info

![ST Crop Issue](https://github.com/SillyTavern/SillyTavern/assets/61097619/a5122dca-e49d-421e-a986-39f8822d9465)

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You've checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Cohee1207** [comment]
You may enable "Never resize avatars" in the User Settings to basically enable the "Free mode" you're mentioning.

**Comment by Sqrlly** [comment]
Ah thanks I did not see that!  I did find the Rectangular option that helped some, but not completely.

Okay so I've set that.  For my Persona when I choose an image from the drive it still forces me to select a 2x3 cropping of the image.  Same thing when I try manually selecting an image for a character:

![image](https://github.com/SillyTavern/SillyTavern/assets/61097619/797c3e1b-a774-4de8-811e-d50223b65bb2)

I haven't tried loading a new character yet, but presuming the option should kick in there for automatic image selecting with a card.
